docs/installation.rst

```diff
@@ -29,7 +29,14 @@ __ Pip-install-upgrade_
 
 From Source
 -----------
-The following assumes `Poetry`_ is already installed.
+
+It is recommend that source installs are done inside of a Python
+`virtual enviornment <https://docs.python.org/3/tutorial/venv.html>`_.
+The following requires that `Poetry`_ is installed, which can be done with:
+
+.. prompt:: bash
+
+   pip install poetry
```

Comment:
The current "preferred" method is I guess via pipx, given that poetry's [install guide](https://python-poetry.org/docs/#installation) puts that at the front of the list. Maybe we can refer to this link instead of parroting any specific method.